.eslintignore

```diff
@@ -1,4 +1,5 @@
 assets/js/build
+assets/js/jquery/3.5.1/jquery.min.js
```

Comment:
now that we removed jquery we can remove this too right?

Reply:
@zachmargolis  It's still there and required, we just moved it out of _config.yml and inlined it so it does not load on every page since we just need it for the contact form:

https://github.com/18F/identity-site/blob/6bcac7b5f76d00c84420e6e9f015147d35f9af3c/_includes/partners/contact.html#L6